--- yaml-cpp-yaml-cpp-0.7.0/test/CMakeLists.txt.orig	2021-07-10 17:53:22.000000000 +0200
+++ yaml-cpp-yaml-cpp-0.7.0/test/CMakeLists.txt	2022-07-02 08:00:07.984458875 +0200
@@ -2,6 +2,7 @@ find_package(Threads REQUIRED)
 
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 set(BUILD_MOCK ON CACHE BOOL "" FORCE)
+set(INSTALL_GTEST OFF CACHE BOOL "" FORCE)
 set(CMAKE_POLICY_DEFAULT_CMP0048 NEW)
 
 add_subdirectory(
