--- yaml-cpp-yaml-cpp-0.7.0/CMakeLists.txt.orig	2022-07-02 09:37:10.699581186 +0200
+++ yaml-cpp-yaml-cpp-0.7.0/CMakeLists.txt	2022-07-02 09:37:13.156234543 +0200
@@ -127,6 +127,9 @@ set_target_properties(yaml-cpp PROPERTIE
   PROJECT_LABEL "yaml-cpp ${yaml-cpp-label-postfix}"
   DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
 
+# substituted in yaml-cpp-config.cmake
+set(EXPORT_TARGETS yaml-cpp CACHE INTERNAL "export targets")
+
 configure_package_config_file(
   "${PROJECT_SOURCE_DIR}/yaml-cpp-config.cmake.in"
   "${PROJECT_BINARY_DIR}/yaml-cpp-config.cmake"
